doc/Developing.md

```diff
@@ -276,6 +277,25 @@ If you have accidentally installed the wrong `yarn` version, you have to remove
 [all-in-one packages]: https://github.com/felixrieseberg/windows-build-tools
 [bug]: https://github.com/theia-ide/theia/issues
 
+### macOS
+
+You need to have the xcode command line tools installed in order to build and run Theia. You can install the tools by running
```

Comment:
@daveyc123 I think it makes sense for xcode, but about node-gyp it looks more like a workaround because we broke something with latest changes or you use outdated versions of npm/node.

Could you open a separate PR for xcode troubleshooting?